README.md

```diff
@@ -1,27 +1,51 @@
 
-CZI is working on a pilot project to offer the entirety of cellxgene’s standardized single cell data resource, containing over 30 million cells, as a set of SOMA-backed objects. This resource, paired with the Python and R SOMA APIs bindings, will enable scientists to query, slice, and stream a subset of the data for analysis in downstream single cell toolkits.
+To address these emerging needs in the single cell ecosystem, CZI in partnership with TileDB is:
 
-Over the next few months, as the SOMA data model and API definition work mature, this resource will be offered to the public with accompanying notebooks demonstrating how to make use of the resource.
 
-**3. Supplementary domain specific libraries and schemas.**
+1. Driving the development of SOMA.
+2. Providing its first implementation, [TileDB-SOMA](https://github.com/single-cell-data/TileDB-SOMA) which utilizes the [TileDB Embedded](https://github.com/TileDB-Inc/TileDB) engine.
+3. Adopting TileDB-SOMA at [CZ CELLxGENE Discover](https://cellxgene.cziscience.com/) to build the [Cell Census](https://github.com/TileDB-Inc/TileDB) which provides efficient access and querying of its data from nearly 50 million cells, compiled from 700+ datasets.
 
-While SOMA is intended to be general purpose, two efforts are underway to build on top of the core SOMA format to make its use more single cell domain adaptable.
+The `SOMA` specification and its `TileDB-SOMA` implementation provides the following capabilities for single-cell data:
 
-The first effort is a set of schemas defining how to capture both multimodal and unimodal single cell data, which will be used as the basis for cellxgene data available via the SOMA API. From the outset, SOMA’s APIs will be designed to interface with multimodal data; the API will enable querying and slicing multimodal datasets along any schema-defined axis.
+1. An abstract specification with flexibility for data from multiple modalities (e.g. RNA, spatial, epigenomics)
+1. A format to store and access datasets larger than memory, as compared to the current paradigm of `.h5ad`/`.mtx`/`.tgz`/`.RData`/`h5Seurat `/ etc.
+1. Eliminates in-memory limitations by providing query-ready data management for reading and writing at low latency and cloud scale. 
+1. R and python APIs with the flexibility to expand to other future languages.
 
-The second effort is a supplemental library called SOMA.io that will enable users to convert SOMA-backed objects to and from the two most popular domain-specific formats: anndata and Seurat.
 
-Both these efforts will be released in the upcoming months in parallel to the two projects above.
+## Developer information
+
+* [SOMA Abstract specification](https://github.com/single-cell-data/SOMA/blob/main/abstract_specification.md) — language-agnostic SOMA API specification.
```

Comment:
No need to upper-case Abstract and lower-case specification -- please lower-case them both